doc/user_guide/samfire.rst

```diff
@@ -0,0 +1,149 @@
+SAMFire
+*******
+
+.. versionadded:: 1.0
+    SAMFire
+
+SAMFire (Smart Adaptive Multi-dimensional Fitting) is an algorithm created to
+avoid the starting value (or local / false minima) problem, which often arises
+when fitting multi-dimensional datasets.
+
+The algorithm will be described in full when accompanying paper is published,
+however we make the implementation available before that.
```

Comment:
"The algorithm will be described in full when the accompanying paper is published,
but we are making the implementation available now."
...or similar